cafebabel/migrations.py

```diff
@@ -0,0 +1,42 @@
+import json
```

Comment:
Wondering about the name of this file as we will need to deal with regular migrations for database changes in the near future

Reply:
I spent quite some time trying to name it for that reason, any suggestion appreciated 😅

Reply:
What about an explicit name like `django-migrations.py`?

Reply:
Good idea 👍